lib/base/threadpool.hpp

```diff
@@ -89,8 +91,9 @@ class ThreadPool
 private:
 	boost::shared_mutex m_Mutex;
 	std::unique_ptr<boost::asio::thread_pool> m_Pool;
-	size_t m_Threads;
 	Atomic<uint_fast64_t> m_Pending;
+
+	void InitialisePool();
```

Comment:
Not sure why you've changed the spelling over my suggestion. You may prefer this spelling, but our code base has a clear favorite:
```console
$ git grep -i initialise | wc -l
0
$ git grep -i initialize | wc -l
444
```

Reply:
Oh, I thought yours was wrong, but it's actually US vs. GB 🐘